granulate_utils/containers/docker.py

```diff
@@ -35,15 +37,27 @@ def get_runtimes(self) -> List[str]:
         return ["docker"]
 
     @staticmethod
-    def _create_container(container: docker.models.containers.Container) -> Container:
+    def _parse_docker_ts(ts: str) -> Optional[datetime]:
+        assert ts.endswith("Z")  # assert UTC
+        if ts.startswith("0001"):
```

Comment:
What? You're expecting some magic value here, please comment it here

Reply:
Added a comment, but in docker engine the timestamps are non-null strings so the zero value is 0001-01-... timestamp